src/applications/personalization/dashboard/components/IconCTALink.jsx

```diff
@@ -60,15 +58,16 @@ const IconCTALink = ({
             toggleName={Toggler.TOGGLE_NAMES.myVaNotificationDotIndicator}
           >
             <Toggler.Enabled>
-              <span
+              <svg
```

Comment:
If there is an icon missing from `va-icon` that you need, please [request it here](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/new?assignees=caw310&labels=platform-design-system-team%2C+experimental_design&projects=&template=3_experimental_design_request.md&title=Experimental+Design+%5Bcomponent+or+pattern+name%5D).